docs/markdown/series.md

```diff
@@ -17,63 +17,63 @@ Type: `Array<Object>`
 Array of data for the series.
 
 #### x
-Type: `number|Object`  
```

Comment:
removing white space is great, but maybe this could be reserved for a single white space removal PR so that this wouldn't cause potential collisions